apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -212,12 +211,10 @@ export class SendMessageEmail extends SendMessageBase {
     }
 
     try {
-      const organization = await this.getOrganization(command.organizationId);
-
       const i18nInstance = await this.initiateTranslations(
         command.environmentId,
         command.organizationId,
-        command.payload.subscriber?.locale || organization?.defaultLocale
+        subscriber.locale
```

Comment:
Aligning the logic here with what we have in [Push](https://github.com/novuhq/novu/blob/next/apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts#L86), SMS, and Chat channel send-message use-cases.